### DISCUSSION TITLE
Uncaught TypeError: Cannot read properties of null (reading 'crossOrigin')

### DISCUSSION THREAD

**Discussion by RomanistHere** [vidstack/player#813] [Q&A]
![image](https://user-images.githubusercontent.com/49812531/233617845-9a5f06bb-3919-4cc6-a4b3-e949007aa547.png)
![image](https://user-images.githubusercontent.com/49812531/233617945-429c47a1-aa06-4d8a-a992-215b4c02c398.png)

It's firing on destroy, what do I do to handle it?

**Comment by mihar-22** [comment]
Resolved in `0.4.5` 👍
